src/main/java/org/octopusden/releng/versions/VersionRange.java

```diff
@@ -1,236 +1,12 @@
 package org.octopusden.releng.versions;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Objects;
-
 /**
  * Simple implementation maven based version range specification.
```

Comment:
This is not an implementation. Move the comment to VersionRangeImpl.

Reply:
This block exists in VersionRangeImpl